### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by Jochen-Kall**
Implemented the changes we discussed in the sync call.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,89 @@
-# ELISA Automotive WG - Navigation and file locations
-## Mailing list, Calendar etc
+# ELISA Automotive WG
+## Mission statement
+The automotive workgroup discusses the conditions and prerequisites the automotive sector needs to integrate Linux into a safety critical system.
+ 
+We focus on actual use cases from the Automotive domain to derive the technical requirements to the kernel as a basis for investigation within the Architecture Workgroup and to serve as a blueprint for actual projects in the future.
```
Comment (vstarwalt) [inline README.md:5]: Change "Architecture Workgroup" to "Automotive Workgroup"?

**Review by vstarwalt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pahmann) [reply README.md:5]: Safety Architecture Workgroup will make it clearer.
Other than this Architecture will remain, as it illustrates the interaction.

Comment (pahmann) [reply README.md:5]: @Jochen-Kall  are you willing to update just with the Safety in front of Architecture Workgroup?

Comment (Jochen-Kall) [reply README.md]: Sure, and done :)

**Pull request merged**